ampd/src/lib.rs

```diff
@@ -40,6 +41,7 @@ mod health_check;
 mod json_rpc;
 mod mvx;
 mod queue;
+pub(crate) mod starknet;
```

Comment:
why `pub(crate`, which is unlike the other modules?

Reply:
I believe we used it somewhere before, but it doesn't need to be pub(crate) anymore. 
My bad. Removing it.